### ISSUE TITLE
Disable Warning Message(s) in case of Targets not being resolvable

### ISSUE THREAD

**Issue by eaglerainbow** [promregator/promregator#164]
## Environment
Promregator 0.7.3 on CF using docker.
Original trigger: @warlockza 

## Summary / Problem Statement
Consciously unprecise target configuration may leave cases where an org/space is empty (no apps installed), but still are configured as target. 
There are also other cases where a resolution isn't possible

## Observed Behavior
We create too many warnings. They may pose an operational problem.

## Expected Behavior
The user is able to disable these type of warnings explicitly. However, it shall be possible to still receive other warnings from `reactivetargetresolver`. 

## Log of discussion

```
Question
If I specify an org and space that exist
But it does not contain any apps
And I have specified an app name
How do you expect promregator to behave
[...]

I am seeing lots of errors in the logs

[...]

is it "Application id could not be found for org '%s', space '%s' and application '%s'. Check your configuration; skipping it for now"?

Yes, that

that's line 264 (of reactivetargetresolver)
```

**Comment by eaglerainbow** [comment]
For the sake of completeness: will be made available to 0.8.0 (and later) as well as 0.7.4 (and later) - not released yet.

**Comment by eaglerainbow** [comment]
[0.7.4](https://github.com/promregator/promregator/releases/tag/v0.7.4) is released.
